ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/versions/capella/block/BlockProcessorCapella.java

```diff
@@ -133,12 +167,98 @@ public void processBlsToExecutionChangesNoValidation(
     }
   }
 
+  // process_withdrawals
+  public void processWithdrawals(
+      final MutableBeaconState genericState, final Optional<ExecutionPayload> maybePayload)
+      throws BlockProcessingException {
+    final MutableBeaconStateCapella state = MutableBeaconStateCapella.required(genericState);
+    final List<Withdrawal> expectedWithdrawals = getExpectedWithdrawals(state);
```

Comment:
nit: I'd push this down a line so that we do the various boiler plate stuff of getting capella versions and then start on the actual logic.